sentinel-adapter/sentinel-spring-webmvc-adapter/README.md

```diff
@@ -0,0 +1,76 @@
+# Sentinel Spring MVC Interceptor
+
+Sentinel provides Spring MVC Interceptor integration to enable flow control for web requests, And support url like '/foo/{id}'
+
+Add the following dependency in `pom.xml` (if you are using Maven):
+
+```xml
+<dependency>
+    <groupId>com.alibaba.csp</groupId>
+    <artifactId>sentinel-spring-webmvc-adapter</artifactId>
+    <version>x.y.z</version>
+</dependency>
+```
+
+Config interceptor
+
+```java
+@Configuration
+public class InterceptorConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        //Add sentinel interceptor
+        addSpringMvcInterceptor(registry);
+
+        //If you want to sentinel the total flow, you can add total interceptor
+        addSpringMvcTotalInterceptor(registry);
+    }
```

Comment:
In my opinion, this pr supports the coexistence of two interceptors, `SentinelSpringMvcTotalConfig ` and `SentinelSpringMvcConfig ` use same `Context name`, but the `SentinelSpringMvcTotalConfig ` is the correct way before `SentinelSpringMvcConfig `.

Reply:
SentinelTotalInterceptor can before SentinelInterceptor, we can use the same context name in every module, see module: sentinel-web-servlet , CommonFilter CommonTotalFilter

Reply:
Can you provide your detailed analysis? thank you.

Reply:
Your design as well, I understand the mistake.  🤣

Reply:
Any other questions? Please advise, thank you!

Reply:
No, I have learned a lot from reading your pr.👍